backend/infrahub/proposed_change/tasks.py

```diff
@@ -294,7 +291,7 @@ async def run_generators(
         for generator in generators
     ]
 
-    diff_summary = await get_diff_summary_cache(pipeline_id=model.branch_diff.pipeline_id, cache=service.cache)
+    diff_summary = await get_diff_summary_cache(pipeline_id=model.branch_diff.pipeline_id, cache=await get_cache())
```

Comment:
Could also be a followup but based on the change it seems we should remove the cache parameter from get_diff_summary_cache and instead call await get_cache() within that function.